tasks.ps1

```diff
@@ -35,9 +32,16 @@ Set-Alias -Name ce -Value CommandAliasFunction -Scope script
 
 [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUserDeclaredVarsMoreThanAssignments', '', Scope = 'Function')]
 $variantApiDeployYamlPath = [System.IO.Path]::GetFullPath((Join-Path ${RepositoryRoot} ".variant/deploy/"))
-$deployYamlsFound = Get-ChildItem -Path $variantApiDeployYamlPath -Filter "*.yaml" -Recurse
-if ($deployYamlsFound.Count -gt 0)
-{
+$testPath = Test-Path -Path $variantApiDeployYamlPath
+Write-Information "Deploy folder exists in .variant directory: $testPath"
+if ($testPath -eq $true) {
+  $deployYamlsFound = Get-ChildItem -Path $variantApiDeployYamlPath -Filter "*.yaml" -Recurse
+}
+else {
+  throw "Deploy folder does not exists"
```

Comment:
Can you make it do the error message informs them where to have a deploy folder. I.e. .variant/deploy

Reply:
Added.